internal/slicer/slicer.go

```diff
@@ -239,6 +246,11 @@ func Run(options *RunOptions) (*Report, error) {
 			if err != nil {
 				return nil, err
 			}
+
+			// Do not add paths with "until: mutate"
+			if pathInfo.Until == setup.UntilMutate {
+				continue
```

Comment:
In this case where the `report.Add` is a couple of lines, I would prefer to have it inside the if statement, that way we can add things to the loop later without worrying about the other of early returns.

Reply:
Makes sense.